CHANGELOG.md

```diff
@@ -34,8 +34,9 @@ const device = new Device(token, {
   tokenRefreshMs: 30000,
 });
 
-device.on('tokenWillExpire', () => {
-  return getTokenViaAjax().then(token => device.updateToken(token));
+device.on('tokenWillExpire', (dev: Device) => {
```

Comment:
Why do we still need to pass the `device`'s reference to the handler if the user already have access to it?

Reply:
It's not strictly necessary, but it matches the rest of the event signatures if we do pass the device to the callback.

Reply:
I'd remove it from the example though, seems like we're promoting an antipattern